### PR TITLE
#210 changed plankton version to use

### PR DESCRIPTION
**Pull request by ChristianLohmann**

### CODE REVIEW

File: test-userdata.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ application_id: my-http-test-app-ami
 application_version: "1.1"
```
Comment (mrandi) [inline test-userdata.yaml]: version should be dynamic, something like that
```application_version: "{{Arguments.ImageVersion}}"```
(and be registered in yourturn)

File: test-userdata.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ application_id: my-http-test-app-ami
...
 
 runtime: Docker
-source: registry.opensource.zalan.do/stups/plankton:cd13
+source: registry.opensource.zalan.do/stups/plankton:cd14
```
Comment (mrandi) [inline test-userdata.yaml]: This could also be dynamic:
```source: registry.opensource.zalan.do/stups/plankton:{{Arguments.ImageVersion}}```

**Comment by ChristianLohmann** [comment]
adapted app name and version to the real plankton name and version

**Comment by mrandi** [comment]
👍

**Pull request merged**